scripts/CEE/kubelet-restart/README.md

```diff
@@ -0,0 +1,13 @@
+# Restart Kubelet on Master Node
```

Comment:
How about generic "Restart Kubelet on a Node"

Reply:
Changes made for it to not be node-specific.! Thanks!